src/index.ts

```diff
@@ -1,9 +1,18 @@
+/**
+ * Main
+ */
 export * from './types';
 export * from './contract';
-export * from './wallet';
-export * from './starknet';
-
-export { pedersen } from './utils/hash';
+export * from './provider';
+export * from './signer';
 
+/**
+ * Utils
+ */
 export * as constants from './constants';
-export * as utils from './utils';
+export * as encode from './utils/encode';
+export * as hash from './utils/hash';
+export * as json from './utils/json';
+export * as number from './utils/number';
+export * as starknet from './utils/starknet';
```

Comment:
true .. All it contains is utils which are starknet specific, but there's no strong need for the developer to use it.

Reply:
maybe we should rename the utils package

Reply:
Haha I agree. I always avoid using names like `utils`, `tools`, `helper` because they don't mean anything. If you wanna sound enterprisey you can call them `services` 🤓

Reply:
Or separate between `services` for business stuff and `infrastructure` for non-business and technical stuff

Reply:
I'll rename it to `stark` as these functions are stark ec specific